pkg/endpoint/bpf.go

```diff
@@ -1156,14 +1208,15 @@ func (e *Endpoint) ApplyPolicyMapChanges(proxyWaitGroup *completion.WaitGroup) e
 	return err
 }
 
-// applyPolicyMapChanges applies any incremental policy map changes
-// collected on the desired policy.
-func (e *Endpoint) applyPolicyMapChanges() error {
+// applyPolicyMapChangesLocked applies any incremental policy map changes
+// collected on the desired policy. The method must be called with the
+// endpoint mutex held. It returns two special errors that must be considered,
+// "ErrPolicyEntryMaxExceeded".
```

Comment:
Nit: the comment says: "it returns two special errors that must be considered" but only one is listed.